apps/status-api/__test__/controllers/OverallStatusController.test.ts

```diff
@@ -0,0 +1,137 @@
+import { StatusApiTesting } from '../../testing/StatusApiTesting'
+import { ApiPagedResponse, WhaleApiClient } from '@defichain/whale-api-client'
+import { WhaleApiProbeIndicator } from '../../src/modules/WhaleApiModule'
+import { HealthIndicatorResult, HealthIndicatorStatus } from '@nestjs/terminus'
+import { Block } from '@defichain/whale-api-client/dist/api/Blocks'
+
+describe('AggregateController - Status test', () => {
+  const apiTesting = StatusApiTesting.create()
+  beforeAll(async () => {
+    await apiTesting.start()
+  })
+
+  afterAll(async () => {
+    await apiTesting.stop()
+  })
+
+  it('/overall - should get operational when Ocean and blockchain are up', async () => {
+    jest
+      .spyOn(apiTesting.app.get(WhaleApiProbeIndicator), 'liveness')
+      .mockReturnValueOnce(getWhaleStatus('up'))
+    jest
+      .spyOn(apiTesting.app.get(WhaleApiProbeIndicator), 'readiness')
+      .mockReturnValueOnce(getWhaleStatus('up'))
+    jest
+      .spyOn(apiTesting.app.get(WhaleApiClient).blocks, 'list')
+      .mockReturnValueOnce(getBlockResponseWithPresetTime(25))
```

Comment:
I don't understand these magic numbers: 25, 36, and 46. Mind to enlighten?

Reply:
The status of the blockchain is determined according to this. 
If a block is mined:
- `< 30 mins`: operational
- `30-45 mins`: degraded
- `> 45 mins`: outage

Reply:
In this case I think its better to add some comments to give more context to the tests.